src/Sulu/Bundle/MediaBundle/Content/Types/MediaSelectionContentType.php

```diff
@@ -56,16 +60,23 @@ class MediaSelectionContentType extends ComplexContentType implements ContentTyp
      */
     private $permissions;
 
+    /**
+     * @var PropertyMetadataMinMaxValueResolver|null
+     */
+    private $propertyMetadataMinMaxValueResolver;
+
     public function __construct(
         MediaManagerInterface $mediaManager,
         ReferenceStoreInterface $referenceStore,
         RequestAnalyzerInterface $requestAnalyzer = null,
-        $permissions = null
+        $permissions = null,
+        PropertyMetadataMinMaxValueResolver $propertyMetadataMinMaxValueResolver = null
```

Comment:
If we make it nullable we need to prepend the type with `?`, because IIRC this will trigger a deprecation in PHP 8.